string[] names
float32[] l2_distances  # the lower, the more probable 'names[i]' is correct

# ROI
uint32 x
uint32 y
uint32 width
uint32 height

uint32 age
bool gender_is_male
float32 gender_confidence
